change/@ni-nimble-components-b44508aa-2a65-4419-8047-367e63603ea2.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@mollykreis looks like some of the stories are clipped on the edge like we saw before for error text in the last row.
Looks good otherwise, feel free to approve stories and merge after addressed.